synapse/models/inet.py

```diff
@@ -825,62 +909,235 @@ def getModelDefs(self):
+                            'ro': 1,
+                            'doc': 'The web account associated with the action.'
+                        }),
+                        ('acct:site', ('inet:fqdn', {}), {
+                            'ro': 1,
+                            'doc': 'The site or service associated with the account.'
+                        }),
+                        ('acct:user', ('inet:user', {}), {
+                            'ro': 1,
+                            'doc': 'The unique identifier for the account.'
+                        }),
+                        # FIXME missing json
+                        #('info', ('json', {}), {
+                        #    'doc': 'Any other data associated with the action.'
+                        #}),
+                        ('time', ('time', {}), {
+                            'doc': 'The date and time the account performed the action.'
+                        }),
+                        ('ipv4', ('inet:ipv4', {}), {
+                            'doc': 'The source IPv4 address of the action.'
+                        }),
+                        ('ipv6', ('inet:ipv6', {}), {
+                            'doc': 'The source IPv6 address of the action.'
+                        }),
+                    )),
 
-                    ('inet:wifi:ap', {}, (
-                        ('ssid', ('inet:wifi:ssid', {}), {
-                            'doc': 'The SSID for the wireless access point.'
+                    ('inet:web:file', {}, (
+                        ('acct', ('inet:web:acct', {}), {
+                            'ro': 1,
+                            'doc': 'The account that owns or is associated with the file.'
                         }),
-                        ('bssid', ('inet:mac', {}), {
-                            'doc': 'The MAC address for the wireless access point.'
+                        ('acct:site', ('inet:fqdn', {}), {
+                            'ro': 1,
+                            'doc': 'The site or service associated with the account.'
+                        }),
+                        ('acct:user', ('inet:user', {}), {
+                            'ro': 1,
+                            'doc': 'The unique identifier for the account.'
+                        }),
+                        ('file', ('file:bytes', {}), {
+                            'ro': 1,
+                            'doc': 'The file owned by or associated with the account.'
+                        }),
+                        # FIXME missing file:base
+                        #('name', ('file:base', {}), {
+                        #    'doc': 'The name of the file owned by or associated with the account.'
+                        #}),
+                        ('posted', ('time', {}), {
+                            'doc': 'The date and time the file was posted / submitted.'
+                        }),
+                        ('ipv4', ('inet:ipv4', {}), {
```

Comment:
this feels like it rolls up into `inet:client`

Reply:
agreed

Reply:
Yea, this ( and any other :ipv4 secondary property that really means "the client" should become inet:client types.  Good call.